app/src/main/java/org/fossasia/susi/ai/chat/ParseSusiResponseHelper.kt

```diff
@@ -76,6 +77,12 @@ class ParseSusiResponseHelper {
                 webSearch = ""
             }
 
+            Constant.STOP -> try {
+                stop = susiResponse.answers[0].actions[1].type
+            } catch (e: Exception) {
+
```

Comment:
Why not logging the exception

Reply:
@iamareebjamal it was not done in any other cases previously

Reply:
if this should be done should it not be for all the cases?

Reply:
Create an issue to log all caught exceptions. Generic exceptions should not even be caught actually

Reply:
@iamareebjamal created a new issue for that purpose #1314